lighthouse-core/audits/seo/plugins.js

```diff
@@ -146,7 +146,7 @@ class Plugins extends Audit {
 
         return {
           source: {
-            type: /** @type {'node'} */ ('node'),
+            ...Audit.makeNodeItem(plugin.node),
             snippet: `<${tagName}${attributes}>${params}</${tagName}>`,
```

Comment:
> sometimes very long snippets

examples?

Reply:
> Should it use the default NodeDetails snippet instead of its custom one?

Maybe, if we wanted to we could parameterize `getNodeDetails/Snippet` to accept a list of attribute names (give it `SOURCE_PARAMS`).

Reply:
> > Does it work well for this audit (with these sometimes very long snippets)?
> 
> examples?

unlike the `getNodeDetails().snippet`, the plugin `attributes` and `params` aren't ever truncated in number or string length, so it could be as long as the original page wanted

> > Should it use the default NodeDetails snippet instead of its custom one?
> 
> Maybe, if we wanted to we could parameterize `getNodeDetails/Snippet` to accept a list of attribute names (give it `SOURCE_PARAMS`).

yeah, though I guess do we care about the `<param>` elements or are the element attributes (plus all the other node details stuff) sufficient to identify it?

Reply:
ehhh let's just use the default snippet so we can keep this simple. it'll be more identifiable now with the devtools node path and everything else, and truncation will prevent the worse cases.